src/java.base/share/classes/java/net/InetAddress.java

```diff
@@ -814,7 +822,7 @@ public String getCanonicalHostName() {
     }
 
     /**
-     * Returns the hostname for this address.
+     * Returns the fully qualified domain name for this address.
```

Comment:
Maybe in the followup issue we could change "this address" to "the given address" considering it's a static method. It's not part of the spec regardless, so only a minor nit.

Reply:
Hello Michael, that's a good catch. I've now pushed an update in this PR which fixes that.